src/main/java/com/exasol/errorreporting/PlaceHoldersFiller.java

```diff
@@ -0,0 +1,85 @@
+package com.exasol.errorreporting;
+
+import java.util.Map;
+
+import com.exasol.errorreporting.PlaceHolderIterator.PlaceHolder;
+
+/**
+ * Class for filling the place holders of texts.
+ */
+class PlaceHoldersFiller {
```

Comment:
```suggestion
class PlaceholdersFiller {
```

i think placeholder is one word ;-)

Reply:
If you agree also correct the other ones